### ISSUE TITLE
Does not work with Facebook new UI

### ISSUE THREAD

**Issue by ngudulalose** [dracula/fb-messenger#15]
Dracula theme doesn't work with Facebook's new UI

**Comment by ppartarr** [comment]
This project uses the obfuscated/minimized css class names from Facebook's UI. Example from the source code:
```
 /* Messenger Header Bar*/
._1tqi
  color: $messengerblue
```

As Facebook has released a new UI, every class has now changed. Example from new Facebook UI (using dev tools):
```
hybvsw6c cjfnh4rs
```

This means that this project has to be completely re-written for the new UI. This is quite a large task and I'm currently very busy so I will not work on this until the new year, sorry for the inconvenience...

I know that Facebook has released a dark mode for facebook.com, hopefully they will also release it for messenger.com. I'll keep you posted in this issue when the I have repaired the theme!

**Comment by nem8** [comment]
It should be removed from Dracula as it is abandoned.

**Comment by zenorocha** [comment]
Can you confirm that this should be deprecated @ppartarr?

**Comment by ppartarr** [comment]
yes, I removed the extension from the chrome store. The project requires a re-write since the messenger UI update. This dracula theme should be archived unless it finds a new maintainer
